README.md

```diff
@@ -30,7 +30,19 @@ Generally speaking, you will use this in combination with the [laminas/laminas-c
 
 ## Usage
 
+Create `.github/workflows/continuous-integration.yml` file with following content:
+
 ```yaml
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - '[0-9]+.[0-9]+.x'
```

Comment:
This may not apply to all projects out there.
I'd say remove this line from here to avoid issues in other projects.

Reply:
How about `on: [push, pull_request]`